src/chrome/content/rules/Eloqua.xml

```diff
@@ -1,4 +1,9 @@
+
 <!--
+Disabled by https-everywhere-checker because:
+Fetch error: http://images.marketing.eloqua.com/ => https://s33.t.eloqua.com/: Too many redirects while fetching 'https://s33.t.eloqua.com/'
```

Comment:
I assume only the root page loops, and there are resources under different paths that don't? We should specify test URLs for those paths and an exclusion for the root page. Leaving disabled for now.